kie-wb/kie-wb-webapp/src/main/resources/org/kie/workbench/KIEWebapp.gwt.xml

```diff
@@ -76,6 +76,9 @@
   <inherits name="org.kie.workbench.common.services.KieWorkbenchCommonServicesAPI"/>
   <inherits name='org.kie.workbench.common.services.refactoring.KieWorkbenchCommonRefactoringAPI'/>
 
+  <!-- Drools Workbench Services -->
+  <inherits name="org.drools.workbench.services.verifier.api.VerifierApi"/>
```

Comment:
Do we need to inherit this here? If you inherit it in [DroolsWorkbenchGuidedDecisionTableEditorClient](https://github.com/droolsjbpm/drools-wb/blob/master/drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/resources/org/drools/workbench/screens/guided/dtable/DroolsWorkbenchGuidedDecisionTableEditorClient.gwt.xml) we should not need it here.